auth/api/iam/metadata_test.go

```diff
@@ -19,20 +19,22 @@
 package iam
 
 import (
+	"github.com/nuts-foundation/nuts-node/core"
 	"github.com/stretchr/testify/assert"
 	"net/url"
 	"testing"
 )
 
+var vpFormats = map[string]map[string][]string{
```

Comment:
let's just refer to the global var

Reply:
what is a good way to prevent an accidental typo changing the global var? Besides code review.

Reply:
not much. A wrong constant would show up in the development network at runtime.